content/docs/connect/connection-pooling.md

```diff
@@ -53,4 +53,39 @@ The **Connection Details** widget on the Neon **Dashboard** provides **Pooled co
 
 Neon uses PgBouncer in _transaction mode_, which does not support Postgres features such as prepared statements or [LISTEN](https://www.postgresql.org/docs/15/sql-listen.html)/[NOTIFY](https://www.postgresql.org/docs/15/sql-notify.html). For a complete list of limitations, refer to the "_SQL feature map for pooling modes_" section in the [pgbouncer.org Features](https://www.pgbouncer.org/features.html) documentation.
 
+## Neon PgBouncer configuration settings
+
+This section describes Neon's PgBouncer configuration. The settings are not user-configurable.
+
+```ini
+[pgbouncer]
+listen_port=6432
```

Comment:
I'm not sure that we need to reveal these settings

```
listen_port
listen_addr
auth_type
auth_user
admin_users
auth_dbname
```

Reply:
removed. Also removed the sslmode settings. We can add them back if necessary.